<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XlabAppMainWindowClass</class>
 <widget class="QMainWindow" name="XlabAppMainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1349</width>
    <height>878</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XlabAppMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_1">
      <item row="0" column="1">
       <widget class="QVTKWidget" name="toprightvtk"/>
      </item>
      <item row="1" column="1">
       <widget class="QVTKWidget" name="bottomrightvtk"/>
      </item>
      <item row="0" column="0">
       <widget class="QVTKWidget" name="topleftvtk"/>
      </item>
      <item row="1" column="0">
       <widget class="QVTKWidget" name="bottomleftvtk"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0" alignment="Qt::AlignTop">
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>230</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTreeView" name="fileNameTree">
         <property name="maximumSize">
          <size>
           <width>211</width>
           <height>211</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="viewButton">
         <property name="text">
          <string>显示</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteDataButton">
         <property name="text">
          <string>删除数据</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1349</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>打开</string>
    </property>
    <addaction name="openDICOM"/>
    <addaction name="openNII"/>
    <addaction name="separator"/>
    <addaction name="exitXlabApp"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="showCross"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openDICOM">
   <property name="text">
    <string>打开DICOM数据</string>
   </property>
  </action>
  <action name="openNII">
   <property name="text">
    <string>打开NII数据</string>
   </property>
  </action>
  <action name="showCross">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>打开/关闭十字线</string>
   </property>
   <property name="iconText">
    <string>打开/关闭十字线</string>
   </property>
   <property name="toolTip">
    <string>打开/关闭四视图十字</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="exitXlabApp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出XlabApp</string>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>c</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="XlabAppMainWindow.qrc"/>
 </resources>
 <connections/>
</ui>
